docs/tutorials/developer-journey/level-5/5.2-ICP-ETH-tutorial.md

```diff
@@ -0,0 +1,433 @@
+# 5.2 Deploying an ETH starter project on the IC
+
+## Overview
+
+
+As you explored in a previous tutorial, the Internet Computer is integrated with the Bitcoin network, allowing for smart contracts to seamlessly communicate from the IC to Bitcoin for multi-chain functionality. The ICP ETH native integration is currently in development, and will include extensive capabilities for Ethereum dapps and tokens on the IC. For example, when complete, the ICP ETH integration will include capabilities such as hosting Ethereum dapps entirely on the IC and creating ckETH and ckERC-20 tokens, which can be swapped without gas fees. 
+
+These capabilities will be enabled through a protocol-level ETH integration, which will initially use HTTPS outcalls to interact with Ethereum APIs to securely query and send transactions to the Ethereum network. These HTTPS outcalls will eventually be replaced by an on-chain Ethereum API on the IC, made possible by running full Ethereum nodes on each ICP replica. 
+
+Through the IC's chain-key cryptography feature, the ETH integration will include chain-key tokens, similar to the design of ckBTC. The ETH integration will expand the possibilities of chain-key tokens to include ckETH and ckERC-20 tokens, including ckUSDC and ckUSDT. Lastly, the ETH integration will provide developers the ability to run existing Solidity code and dapps on the IC through an on-chain EVM solution.
+
+:::caution
+As mentioned, the ICP ETH integration is still in development. In the future, a there will be an additional variation of this developer journey series that focuses solely on Ethereum development on the IC.
+:::
+
+For this tutorial, a sample boilerplate project will be explored that showcases the basis of ICP ETH integration, which uses HTTPS outcalls to query information from the Ethereum network. 
+
+## Deploying the ETH starter project
+
+In this tutorial, you'll use the DFINITY IC ETH starter project to deploy a boilerplate dapp that showcases how the IC can query information from the Ethereum network using HTTPS outcalls. This starter dapp is used to verify the ownership of NFTs minted on the Ethereum network, and supports queries to the Ethereum mainnet and the Sepolia and Goerli test networks. 
+
+### Project technology stack
+
+This starter project is comprised of 4 canisters:
+
+- The frontend canister, which uses TypeScript, Vite and React as the core components to create the user interface.
+
+- The backend canister, written in Motoko, that uses the Motoko package manager Mops and mo-dev, a live development server for Motoko. 
+
+- The Ethereum integration canister, which is written in Rust to utilize the `ethers-core` library in order to interact with Ethereum data structures. This canister also includes Metamask, which is a wallet and browser extension used for interacting with Ethereum dapps. 
```

Comment:
@rvanasa @dfx-json Do we plan to update the starter project to use the EVM RPC canister? If so, we probably should do that before creating videos for the dev journey.

Reply:
The video for this tutorial won't be recorded for about 3 months; currently we are recording one video a week, and just recorded episode 1.1 yesterday. At that rate, the video for this tutorial won't be recorded until sometime in March.